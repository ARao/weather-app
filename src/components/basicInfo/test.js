import React from 'react';
import { shallow } from 'enzyme';
import { BasicInfo } from './index';


describe('Basic', () => {
  const weather = {
    "location": {
      "name": "Paris",
      "region": "Ile-de-France",
      "country": "France",
      "lat": 48.87,
      "lon": 2.33,
      "tz_id": "Europe/Paris",
      "localtime_epoch": 1547152354,
      "localtime": "2019-01-10 21:32"
    }
  }
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<BasicInfo weather={weather} />);
  })

  it('++++ render Basic', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('++++ snapshot match ', () => {

    expect(wrapper).toMatchSnapshot();
  });
});
