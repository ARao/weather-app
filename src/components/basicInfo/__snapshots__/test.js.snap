// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic ++++ snapshot match  1`] = `
<div
  className="row"
>
  <div
    className="ml-auto"
  >
    <h2>
      Paris
    </h2>
    <h3>
      France
    </h3>
    <h4>
      <span
        className="pr-2"
      >
        48.87
      </span>
      <span>
        2.33
      </span>
    </h4>
  </div>
</div>
`;
